text/0000-api-evolution.md

```diff
@@ -0,0 +1,748 @@
+- Feature Name: not applicable
+- Start Date: 2015-05-04
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+
+This RFC proposes a comprehensive set of guidelines for which changes to
+*stable* APIs are considered breaking from a semver perspective, and which are
+not.  These guidelines are intended for both the standard library and for the
+crates.io ecosystem.
+
+This does *not* mean that the standard library should be completely free to make
+non-semver-breaking changes; there are sometimes still risks of ecosystem pain
+that need to be taken into account. Rather, this RFC makes explicit an initial
+set of changes that absolutely *cannot* be made without a semver bump.
+
+Along the way, it also discusses some interactions with potential language
+features that can help mitigate pain for non-breaking changes.
+
+The RFC covers only API issues; other issues related to language features,
```

Comment:
In the past, for better or worse, sometimes changes to things like command line arguments to `rustc` were for various reasoned classified as "library issues" (as opposed to language issues).

While the first paragraph does indeed restrict the domain to "standard library" and "crates.io ecosystem", I think this paragraph can stay -- perhaps just modify it to make it clear that this paragraph is in fact a consequence a fact that all these things are not part of the standard library nor crates.io ecosystem.